services/merak-network/utils/constant.go

```diff
@@ -0,0 +1,26 @@
+/*
+MIT License
+Copyright(c) 2022 Futurewei Cloud
+    Permission is hereby granted,
+    free of charge, to any person obtaining a copy of this software and associated documentation files(the "Software"), to deal in the Software without restriction,
+    including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and / or sell copies of the Software, and to permit persons
+    to whom the Software is furnished to do so, subject to the following conditions:
+    The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+    WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+package utils
+
+const CODE_SUCCESS uint16 = 200
+const CODE_FAILED uint16 = 500
+const CODE_NOT_FOUND uint16 = 400
+
+const VPC string = "vpc:"
+const SUBNET string = "subnet:"
+const Router string = "router:"
+const SECURITYGROUP string = "securityGroup:"
+const NETCONFIG string = "netconfig:"
+
+const ALCORURL string = "10.213.43.77"
```

Comment:
@yanmo96 Please create an issue for the hard code ip address of Alcor, in next release we need to use the `service_config` to get these type of address and url.